website/pages/docs/agent/config-entries/service-intentions.mdx

```diff
@@ -0,0 +1,331 @@
+        HTTP {
+          PathPrefix = "/mycompany.BillingService/"
+        }
+      }
+    ]
+  }
+  # NOTE: a default catch-all based on the default ACL policy will apply to
+  # unmatched connections and requests. Typically this will be DENY.
+]
+```
+
+You can mix and match L4 and L7 intentions per source:
+
+```hcl
+Kind = "service-intentions"
+Name = "api"
+Sources = [
+  {
+    Name   = "hackathon-project"
+    Action = "deny"
+  },
+  {
+    Name   = "web"
+    Action = "allow"
+  },
+  {
+    Name = "nightly-reconciler"
+    Permissions = [
+      {
+        Action = "allow"
+        HTTP {
+          PathExact = "/v1/reconcile-data"
+          Methods   = ["POST"]
+        }
+      }
+    ]
+  },
+  # NOTE: a default catch-all based on the default ACL policy will apply to
+  # unmatched connections and requests. Typically this will be DENY.
+]
+```
+
+## Available Fields
+
+- `Kind` - Must be set to `service-intentions`
+
+- `Name` `(string: <required>)` - The destination of all intentions defined in
+  this config entry. This may be set to the wildcard character (`*`) to match
+  all services that don't otherwise have intentions defined.
+
+- `Namespace` `(string: "default")` <EnterpriseAlert inline /> - Specifies the namespace the config entry will apply to.
+  This may be set to the wildcard character (`*`) to match
+  all services in all namespaces that don't otherwise have intentions defined.
+
+- `Meta` `(map<string|string>: nil)` - Specifies arbitrary KV metadata pairs.
+
+- `Sources` `(array<SourceIntention>)` - The list of all [intention sources and 
+  the authorization granted to those sources](#sourceintention). The order of
+  this list does not matter, but out of convenience Consul will always store
+  this reverse sorted by intention precedence, as that is the order that they
+  will be evaluated at enforcement time.
+
+### `SourceIntention`
+
+- `Name` `(string: <required>)` - The source of the intention.
+  For a `Type` of `consul` this is the name of a Consul service. The service
+  doesn't need to be registered.
+
+- `Namespace` `(string: "default")` <EnterpriseAlert inline /> - The namespace
+  for the `Name` parameter.
+
+- `Action` `(string: "")` - For an L4 intention this is required, and should be
+  set to one of `"allow"` or `"deny"` for the action that should be taken if
+  this intention matches a request.
+
+  This should be omiitted for an L7 intention as it is mutually exclusive with
+  the `Permissions` field.
+
+- `Permissions` `(array<IntentionPermission>)` - The list of all [additional L7
+  attributes](#intentionpermission) that extend the intention match criteria.
+
+  Permission precedence is applied top to bottom. For any given request the
+  first permission to match in the list is terminal and stops further
+  evaluation. As with L4 intentions, traffic that fails to match any of the
+  provided permissions in this intention will be subject to the default
+  intention/acl policy.
```

Comment:
```suggestion
  intention/ACL policy.
```

Reply:
I resolved this by actually cribbing a section from the older intentions docs that had a much better blurb.